packages/avro-ts-cli/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@ovotech/avro-ts-cli",
   "description": "A cli to convert avro schemas into typescript interfaces",
-  "version": "1.0.4",
+  "version": "1.1.4",
```

Resetting the patch version

```suggestion
  "version": "1.1.0",
```

---

oh yeah, :man_facepalming: